llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -964,7 +964,8 @@ void VPWidenIntrinsicRecipe::execute(VPTransformState &State) {
   Module *M = State.Builder.GetInsertBlock()->getModule();
   Function *VectorF =
       Intrinsic::getOrInsertDeclaration(M, VectorIntrinsicID, TysForDecl);
-  assert(VectorF && "Can't retrieve vector intrinsic.");
```

Comment:
Is this change still need?

Reply:
Probably sufficient to keep the existing message